tdns/ixfr/ixfr.go

```
@@ -0,0 +1,148 @@
+package ixfr
+
+import (
+	"fmt"
+
+	"github.com/miekg/dns"
+)
+
+var logger = fmt.Printf
```

golangci-lint (1.62.2)

var `logger` is unused

(unused)